config/sources/families/k3.conf

```diff
@@ -10,21 +10,22 @@
 source "${BASH_SOURCE%/*}/include/k3_common.inc"
 
 declare -g LINUXFAMILY="k3"
+declare -g ATFSOURCE="https://github.com/ARM-software/arm-trusted-firmware"
+declare -g TI_PSDK_TAG="11.01.16"
```

Comment:
This var is only used in the vendor case, why not move it down into that case statement.